components/HighlightedLinks/src/index.scss

```diff
@@ -0,0 +1,41 @@
+.denhaag-highlighted-links {
```

Comment:
This component must be edited from wordpress content editor. Therefor it's already within a container and must breakout of the page container, due the background is fullwidth.

My suggestion would be to make this element position:relative, and set the background-color on the after u make 100vw. 

This way the container is always the same width as the parent-container.

Reply:
I removed the container, the tokens and the css related to it.